app/views/developers/_form.html.erb

```diff
@@ -375,7 +375,7 @@
           <h3 id="notifications" class="text-lg font-medium leading-6 text-gray-900"><%= t(".notifications.title") %></h3>
         </div>
         <div class="mt-5 md:mt-0 md:col-span-2">
-          <fieldset>
+          <fieldset class="mb-4">
```

Comment:
Let's remove this and add `space-y-4` to the parent `<div>` element so it will apply even if we add more notification options.